demonstrations/tutorial_qubitization.py

```diff
@@ -0,0 +1,165 @@
+r"""Intro to Qubitization
+=========================
+
+Encoding a Hamiltonian into a quantum computer is a fundamental task for many applications, but the way to do it is not unique. One method that has gained special status is known as **qubitization**. In this demo, we will introduce the qubitization operator and explain how to view it as a rotation operator. This perspective is useful when combining qubitization with `quantum phase estimation <https://pennylane.ai/qml/demos/tutorial_qpe/>`_ (QPE). We will demonstrate this by implementing qubitization-based quantum phase estimation, with code provided using the :class:`~.pennylane.Qubitization` template in PennyLane.
+
+.. figure:: ../_static/demo_thumbnails/opengraph_demo_thumbnails/OGthumbnail_large_qubitization.png
+    :align: center
+    :width: 70%
+    :target: javascript:void(0)
+
+Qubitization operator
+----------------------
+
+Qubitization is a block-encoding technique that, as we will see, is particularly useful for tasks such as estimating eigenvalues, among other applications.
+For a Hamiltonian :math:`\mathcal{H}`, given in its `representation via a linear combination of unitaries (LCU) <https://pennylane.ai/qml/demos/tutorial_lcu_blockencoding/>`_, the qubitization operator is defined as:
```

Comment:
I'm going to delete it as it was too repetitive with the link after it

```suggestion
For a Hamiltonian :math:`\mathcal{H}`, given in its representation via a linear combination of unitaries (LCU), the qubitization operator is defined as:
```